1st Draft of Blueberry Networking Outline

gcwyrostek opened the pull request:
Submitted the 1st draft of our outline. Still awaiting comments from other group members, but looking for feedback.

nfarnan reviewed (changes requested):
You will need to support several goals. This, on its own, is insufficient and underspecified at that. You are a group of 8 people, and all members will need to make significant effort each week to get a working game by the end of the term. Add additional goals and more details on this goal (e.g., what data structures to use? Do you have choices to make and considerations?)

zimores321 commented:
I think I have it fixed, I had a few commits cause the markdown looked wrong

gcwyrostek committed:
Started Blueberry Outline and update topics

Updated presentation details Blueberry and added topics related to networked multiplayer design philosophies.

gcwyrostek committed:
Merge pull request #1 from gcwyrostek/patch-1

Started Blueberry Outline and update topics

gcwyrostek committed:
Update presentation topics in Networking.md

gcwyrostek committed:
Continued work on Blueberry Networking Presentation Outline

Continued work on Blueberry Networking Presentation Outline

nfarnan reviewed (changes requested):
Have about 15 minutes to work with due to repeats within your presentation, and with previous talk

gcwyrostek committed:
Revise networking presentation timings and topics

Updated timing and content for networking presentation.

nfarnan commented:
Combine first two 10min items to 5 min each (or 10 min total combined)

Add 10 minutes to 15 min section

gcwyrostek committed:
Reduce presentation times for networking topics
